src/playground/playground.module.ts

```diff
@@ -142,31 +148,23 @@ export const NB_EXAMPLE_COMPONENTS = [
   NbBadgeShowcaseComponent,
   NbContextMenuShowcaseComponent,
   NbContextMenuClickComponent,
+  NbChipsShowcaseComponent,
+  NbChipsTabbableComponent,
+  NbChipsRemovableComponent,
+  NbChipsColorsComponent,
 ];
 
-export const NB_ENTRY_COMPONENTS = [
-  NbDynamicToAddComponent,
-];
+export const NB_ENTRY_COMPONENTS = [NbDynamicToAddComponent];
 
 @NgModule({
-  imports: [
-    CommonModule,
-    NbPlaygroundRoutingModule,
-    ...NB_MODULES,
-  ],
-  exports: [
-    ...NB_ENTRY_COMPONENTS,
-    ...NB_EXAMPLE_COMPONENTS,
-  ],
+  imports: [CommonModule, NbPlaygroundRoutingModule, ...NB_MODULES],
+  exports: [...NB_ENTRY_COMPONENTS, ...NB_EXAMPLE_COMPONENTS],
   declarations: [
     NbPlaygroundLayoutComponent,
     NbPlaygroundBaseComponent,
     ...NB_ENTRY_COMPONENTS,
     ...NB_EXAMPLE_COMPONENTS,
   ],
-  entryComponents: [
-    ...NB_ENTRY_COMPONENTS,
-  ],
+  entryComponents: [...NB_ENTRY_COMPONENTS],
 })
-export class NbPlaygroundModule {
-}
+export class NbPlaygroundModule {}
```

Comment:
let's roll these style changes back, as they are are not very important and just add noise to the commit, among the other important changes.